vagrant/boxes.d/99-base-dev.yaml

```diff
@@ -0,0 +1,72 @@
+---
+centos7-katello-devel:
+  primary: true
+  box: centos7
+  memory: 12000
+  ansible:
+    playbook: 'playbooks/katello_devel.yml'
+    group: 'devel'
+    variables:
+      ssh_forward_agent: true
+      foreman_devel_github_push_ssh: True
+      katello_devel_github_username: <REPLACE ME>
```

Comment:
This is the reason we kept these devel definitions in an example file, because we dont want users committing changes to configuration options such as this.